### PR TITLE
Implement condition function zero

### PR DESCRIPTION
**Pull request by ErikRichardS**
Add code for function zero along with test functions for various cases.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,4 +29,5 @@ hs_err_pid*
 #Eclipse auto-generated files
 .classpath
 .project
-*core.prefs
\ No newline at end of file
+*.core.prefs
+*.core.resources.prefs
```
Comment (nilsstre) [inline .gitignore]: Can't you combine these two lines into `*.core.*`?

File: src/main/java/decide/DecideHelpFunctions.java [hunk 2]
```diff
@@ -1,9 +1,47 @@
 package decide;
 
 public class DecideHelpFunctions {
-
-    public boolean conditionFunctionZero(double length, double[][] coordinates) {
-        return true;
+	
+	// Equal, Lesser, Greater
+	// Return 1 if a > b, 0 if a = b, -1 if a < b
```
Comment (nilsstre) [inline src/main/java/decide/DecideHelpFunctions.java]: Use proper JavaDoc syntax when commenting functions. See `conditionFunctionSeven` for example.

File: src/main/java/decide/DecideHelpFunctions.java [hunk 3]
```diff
@@ -1,9 +1,47 @@
...
+		if(Math.abs(diff) < 0.000001) {
+			return 0;
+		}
+		if(diff > 0) {
+			return 1;
+		}
+		return -1;
+	}
+	
+	// Calculates the distance between two 2D coordinates
```
Comment (nilsstre) [inline src/main/java/decide/DecideHelpFunctions.java]: Use proper JavaDoc syntax when commenting functions.

**Review by nilsstre — CHANGES_REQUESTED** [review]
The code looks good but I think you should update the method comments for the private methods you added.

Also, you didn't follow the course mandated commit message syntax, that states that commit message have to include both what type of commit it's and the issue number the commit is related to. Yesterday we agreed to start each commit message with `<type>/<issue-number>`. Your commits in should therefore have started with `feat/#28`.

File: .gitignore (hunk 1 above)
Comment (ErikRichardS) [reply .gitignore]: I don't know if there are any other .core. files not related to Eclipse. Might instead change it to *.prefs .

File: src/main/java/decide/DecideHelpFunctions.java [hunk 4]
```diff
@@ -3,9 +3,56 @@
...
+		return dist;
+	}
+	
+	/**
+	 * Takes an array of coordinates and returns true if two consecutive coordinates are at least the length apart
+	 * @param length 	  : The minimum distance between coordinates for function to return true
+	 * @param coordinates : The coordinates being measured
+	 * @return			  : True if two consecutive coordinates are at at least the minimum distance, else false
+	 */
+    public static boolean conditionFunctionZero(double length, double[][] coordinates) {
```
Comment (nilsstre) [inline src/main/java/decide/DecideHelpFunctions.java:46]: Should be `Point2D[] coordinates`

**Review by nilsstre — CHANGES_REQUESTED** [review]

**Pull request closed without merging**